core/src/events.rs

```diff
@@ -330,6 +333,33 @@ impl<'gc> ClipEvent<'gc> {
     }
 }
 
+/// Control inputs to a text field
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum TextControlCode {
+    // TODO: Extend this
```

Comment:
(Nit: Is this `TODO` still relevant?).

Reply:
That was from the original PR I rebased, so not 100% sure TBH

Reply:
Looking again, yes it's still relevant because it's referring to the unimplemented control codes for Home, End, etc